.github/workflows/winget-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Submit Winget Package to Windows Package Manager Community Repository
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: "Specific tag name"
+        required: false
+        type: string
+env:
+  CI: true
+jobs:
+  winget-publish:
+    name: Publish winget package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set winget version env
+        env:
+          TAG_NAME: ${{ inputs.tag_name || github.event.release.tag_name }}
+        run: echo "WINGET_TAG_NAME=$(echo ${TAG_NAME#v})" >> $GITHUB_ENV
+      - name: Submit package to Windows Package Manager Community Repository
+        uses: vedantmgoyal2009/winget-releaser@v2
+        with:
+          identifier: F3d.F3d
```

Comment:
thats a strange name.

F3D is the app
f3d is the project
f3d-app is the org
libf3d is the lib

but F3d is nothing.

Reply:
That's the name of the app that had been submitted to the winget-pkgs repo. What do you think would be a better identifier? `F3D.F3D, f3d.F3D or f3d-app.F3D`? Would need to make a PR to change that identifier before the first run of the action then.

Reply:
indeed. 

Most definitely `f3d-app.f3d`, that would be coherent with other packages.

Reply:
Alright, just made a PR to winget-pkgs https://github.com/microsoft/winget-pkgs/pull/173403